src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private static final int DIVIDER_FOR_AREA = 2;
+    private int upperSide;
+    private int lowerSide;
+    private int height;
+
+    public IsoscelesTrapezoid(String color, int upbothside, int downbothside, int height) {
```

Comment:
The parameter names 'upbothside' and 'downbothside' could be improved for better readability. Consider using 'upperSide' and 'lowerSide' to match the class field names and maintain consistency.